modules/bioinspired/test/test_retina_ocl.cpp

```diff
@@ -134,31 +94,35 @@ TEST_P(Retina_OCL, Accuracy)
 
     Mat gold_parvo;
     Mat gold_magno;
-    ocl::oclMat ocl_parvo;
-    ocl::oclMat ocl_magno;
+    UMat ocl_parvo;
+    UMat ocl_magno;
 
-    for(int i = 0; i < RETINA_ITERATIONS; i ++)
+    for(int i = 0; i < test_loop_times; i ++)
```

Comment:
`test_loop_times` is not suitable here (it is used as counter of random image generation - sizes/ROI)